third_party/egeria/usage/example_transactions.csv.json

```diff
@@ -2,7 +2,7 @@
     "class": "NewCSVFileAssetRequestBody",
     "displayName": "transactions.csv",
     "description": "Synthetic Financial Dataset For Fraud Detection",
-    "fullPath": "{\"data_store\":\"S3\", \"bucket\":\"mybucket\", \"endpoint\":\"s3.eu-de.cloud-object-storage.appdomain.cloud\", \"object_key\":\"transactions.csv\", \"region\":\"eu\", \"data_location\": \"US\"}",
+    "fullPath": "{\"data_store\":\"S3\", \"bucket\":\"mybucket\", \"endpoint\":\"s3.eu-de.cloud-object-storage.appdomain.cloud\", \"object_key\":\"transactions.csv\", \"region\":\"eu\", \"data_location\": \"US\", \"credentials-secret-ref\": 'default/my-secret'}",
     "columnHeaders": [
```

Comment:
@rohithdv- a minor comment (not a blocker for merging)- I just noticed that all other fields contain underscore in their key names so you could consider changing the new field to be in the same format.

Reply:
Fixed. Please check. Thanks.

Reply:
Thanks it works OK.